### ISSUE TITLE
mdSelect causes page to jump to top in firefox

### ISSUE THREAD

**Issue by AkhilNaidu09** [angular/material#10719]
**Actual Behavior**:
mdSelect causes page to jump to top in firefox:
getting this issue in Firefox and as well as in iPad also
Click the mdSelect at the bottom, you will jump back to the top, please fix this issue, it is a big thing to the users of a website.

**CodePen** (or steps to reproduce the issue): *
 -  `https://codepen.io/anon/pen/VLxmyM. 

**AngularJS Versions**: *
 -  `Angular : 1.5.8`
 -  `Material : 1.1.4`

**Additional Information**:
 -  `FireFox and iPad Safari & iPad Chrome: *`
 -  `FireFox v 53.0b1 *`

**Comment by mckenzielong** [comment]
It also has a brief jump on Chrome 58 / Opera 45

**Comment by AkhilNaidu09** [comment]
Work around 

```
_:-ms-input-placeholder, :root {
  body[style] {
    position: inherit !important;
    top: inherit !important;
    width: inherit !important;
  }

  .md-click-catcher.md-wbTheme-theme {
    height: 100%;
    position: fixed;
    width: 100%;
  }
}
```

**Comment by Splaktar** [comment]
Verified that this is working in latest master (used in above CodePen). The fix went into 1.1.5.
